.github/workflows/build_and_test.yml

```diff
@@ -5909,7 +5909,7 @@ jobs:
         cd OpenDDS
         . setenv.sh
         cd tests/DCPS/Messenger
-        perl $ACE_ROOT/bin/mwc.pl -type gnuace -static
+        perl $DDS_ROOT/bin/opendds_mwc.pl -type gnuace -static
```

Comment:
Is `-static` needed or is it already in `opendds_mwc.pl`?  Same for `-type gnuace`.

Reply:
Yes, both are included:

https://github.com/OpenDDS/OpenDDS/blob/62c78ee53f7556b99c517d192fe4fa2d99f394d5/configure#L2530-L2533

Then later:

https://github.com/OpenDDS/OpenDDS/blob/62c78ee53f7556b99c517d192fe4fa2d99f394d5/configure#L2549

Reply:
Also it should be in the PATH, so it could just be: `opendds_mwc.pl`

Reply:
When I peeked at GHA, it looked like `-type gnuace` was there but `-static` was not.  Probably just missed it.

Reply:
> Also it should be in the PATH, so it could just be: `opendds_mwc.pl`

That's right, but also keep in mind that on Windows sometimes the `*.pl` association doesn't work so it needs an explicit `perl %DDS_ROOT%\bin\opendds_mwc.pl`

Reply:
``` my $static = (($opts{'static'} && $is_windows) ||
    ($cross_compile && $buildEnv->{'build'} eq 'host' && $mpctype ne 'gnuace'));```

From this, it seems like `-static` is superfluous on the non-windows builds.